core/src/subgraph/instance_manager.rs

```diff
@@ -230,6 +230,9 @@ impl<S: SubgraphStore> SubgraphInstanceManager<S> {
             (manifest, manifest_idx_and_name)
         };
 
+        let static_filters = self.static_filters
+            || (manifest.network_name() == "bsc" && manifest.templates.len() > 10000);
```

Comment:
I think it would make sense to do this for any network.

Reply:
@leoyvens thoughts about this?